index.js

```diff
@@ -91,6 +99,29 @@ class Store {
     return this;
   }
 
+  /**
+   * Assign `value` to `key` while retaining prior members of `value` if
+   * `value` is a map.
+   *
+   * ```js
+   * store.set('a', { b: c });
+   * //=> {a: { b: c }}
+   *
+   * store.append('a', { d: e });
+   * //=> {a: { b: c, d: e }}
+   * ```
+   *
+   * @name .append
+   * @param {String} `key`
+   * @param {any} `val` The value to append to `key`. Must be a valid JSON type: String, Number, Array or Object.
+   * @return {Object} `Store` for chaining
+   * @api public
+   */
+
+  append(key, val) {
```

Comment:
Maybe this should be called `assign` or `merge`?

Reply:
Renamed to `merge`